<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Administrator tables</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="surveyButton">
        <property name="text">
         <string>Survey</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="surveyTypeButton">
        <property name="text">
         <string>SurveyType</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="transectButton">
          <property name="text">
           <string>Transect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="transectImportButton">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Import</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../images/plus.png</normaloff>../../../images/plus.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="boatButton">
        <property name="text">
         <string>Boat</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="plateformButton">
        <property name="text">
         <string>Plateform</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
